### DISCUSSION TITLE
Installation problem :(

### DISCUSSION THREAD

**Discussion by jjgonc** [directus/directus#6729] [Archive]
Hey, while trying run the command **npx create-directus-project example-project** i couldn't make it to successfully install Directus...
I got the following error: 
![Screenshot from 2021-07-09 09-22-25](https://user-images.githubusercontent.com/61986130/125052309-bd92cf80-e09b-11eb-9d42-b85900fc60c6.png)

Maybe it could be an easy error to solve, but I'm new in this "world" :)

**Comment by Lucas-Gruber** [comment]
Hey, have you already created the DB  "projeto" in MySQl, MariaDB or Aurora?

**Reply by jjgonc** [reply]
Yeah, just managed it! Thanks anyway
